_posts/2024-10-08-Internal-functioning-of-sorting-in-hybrid-query.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title:  "How sorting works in hybrid queries"
+authors:
+- varunudr
+- minalsha
+- vamshin
+- kolchfa
+date: 2024-10-08
+categories:
+  - technical-posts
+meta_keywords: hybrid query, hybrid search, vector search, search, semantic and lexical search
+meta_description: Discover how sorting works in hybrid queries in OpenSearch, from sorting individual subquery results to combining them based on your search criteria.
```

Comment:
meta_keywords: hybrid queries, semantic search, search relevance, query execution workflow, OpenSearch hybrid search 

meta_description: Explore how sorting works in hybrid queries, the query execution workflow, and ways to leverage these sorting capabilities within OpenSearch to improve search relevance and usability of results.